ir-engine/etherealengine-archive
# IR-2653: allow editor to change project settings

## Summary

Update hooks in project settings to allow users with editor permission to change project settings

## Subtasks Checklist

## Breaking Changes

## References
closes #_insert number here_

## QA Steps

In MT go into the project that have been shared with you and you have editor access 
try to update any of the settings to that project. 
it should successfully persists your changes to the settings
